### DISCUSSION TITLE
Community Extensions & Content

### DISCUSSION THREAD

**Discussion by bdbch** [ueberdosis/tiptap#2973] [Community Extensions]
**Hello dear Tiptap community.**

As you may noticed, we enabled a **Stale bot** last week which caused a few issues with general information and discussions to get unpinned. This is why we now have a **Community Extension** category here in [Discussions](https://github.com/ueberdosis/tiptap/discussions).

Feel free to share your custom content here and for everyone else - give feedback, vote your favorite custom content and keep the good moods coming!

If you are looking for the ticket that was used before Discussions, take a look [here](https://github.com/ueberdosis/tiptap/issues/819).

### Why should you publish your solutions outside of the Tiptap repository?

You may wonder - why should my extension not be added to the Tiptap repository?
As the überdosis team is maintaining and supporting the Tiptap repository, everything that lands in the repo needs to be maintained by us. Since custom extensions and packages are outside of our support, we ask you to first publish it as your own library and share it with us.

If you want to include it in Tiptap we'd invite you to discuss this with us so we can take a look at your codebase and make ourselves ready to give support and maintainment for your package as well.

**Comment by sereneinserenade** [comment]
also https://github.com/ueberdosis/awesome-tiptap/#community-extensions has a list of community extensions 😃

**Comment by amirhhashemi** [comment]
`tiptap-text-direction` is my new extension based on an old PR that automatically adds `dir="ltr"` or `dir="rtl"` to a configurable list of nodes based on their content. It's a must-have if you want to support multiple languages.

 https://github.com/amirhhashemi/tiptap-text-direction

**Comment by Easy-Martin** [comment]
https://github.com/Easy-Martin/tiptap_search_replace_plugin
